odoo/addons/base/models/ir_rule.py

```diff
@@ -243,30 +214,35 @@ def _make_access_error(self, operation, records):
         # so it is relatively safe here to include the list of rules and record names.
         rules = self._get_failing(records, mode=operation).sudo()
 
-        records_description = ', '.join(['%s (id=%s)' % (rec.display_name, rec.id) for rec in records[:6].sudo()])
+        records_sudo = records[:6].sudo()
+        company_issue = any('company_id' in (r.domain_force or []) for r in rules)
```

Comment:
This condition always confuses me, as `domain_force` is not a list but a string.  What about deconfusing it?
```suggestion
        company_issue = any('company_id' in (r.domain_force or '') for r in rules)
```

Reply:
Yep, you have perfectly right!